client/src/components/squids/hooks/useSquidList.js

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from "react-query";
+
+import { ApiClient } from "../../../backend/ApiClient";
+
+export const useSquidList = () => {
+  const apiClient = new ApiClient();
+
+  return useQuery("squids", () => apiClient.get("/squids").then((resp) => resp.data), {
```

Comment:
Nicely queried -- I like the staleTime infinity, since this data likely won't be updating frequently